### PR TITLE
docs: translate intro into spanish

### PR DESCRIPTION
**Pull request by lopezator**
**What is the problem I am trying to address?**

There isn't a spanish translation for the intro doc.

**How is the fix applied?**

Translating it into spanish.

**Mention the issue number it fixes or add the details of the changes if it doesn't have a specific issue.**

Fixes #en->es 😅

### CODE REVIEW

File: docs/content/_index.es.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+¡Bienvenidos, Gophers! Estamos deseando explicarte en que consiste Athens...
+
+En esta página web, está documentado Athens en detalle. Te enseñaremos que hace, por que es necesario, y como puedes ejecutarlo por tu cuenta. A continuación te mostramos un breve listado.
+
+#### ¿Qué hace Athens?
+
+Athens proporciona un servidor para [Go Modules](https://github.com/golang/go/wiki/Modules) que puedes ejecutar tú mismo. Almacena código fuente público y privado por tí, para que no tengas que obtenerlo directamente desde un sistema de control de código fuente (VCS) como GitHub o GitLab.
+
+#### ¿Qué hace?
```
Comment (arschles) [inline docs/content/_index.es.md]: Por qué le importa?

File: docs/content/_index.es.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: "Introducción"
+date: 2019-03-14T08:44:12+00:00
+---
+
+<img src="/banner.png" width="600" alt="Athens Logo"/>
+
+## Athens es un servidor para tus dependencias GO
```
[the remark this answers is not in the document]
Comment (lopezator) [reply docs/content/_index.es.md]: Sure!

File: docs/content/_index.es.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+¡Bienvenidos, Gophers! Estamos deseando explicarte en que consiste Athens...
```
[the remark this answers is not in the document]
Comment (lopezator) [reply docs/content/_index.es.md]: Both would correct depending on the context, it just sounds better "deseando" for me in this specific case (at least in Spanish from Spain), no idea about latam.

See:

https://www.linguee.es/ingles-espanol/traduccion/we+are+looking+forward+to.html

File: docs/content/_index.es.md (hunk 1 above)
Comment (lopezator) [reply docs/content/_index.es.md]: What you wrote in Spanish literally would mean:

"None of your business!" 😅

Of you want to preserve the original meaning of "Why it matters", although this construction is not as common in Spanish I suppose you could use:

¿Por qué tiene tanta importancia?

Comment (lopezator) [reply docs/content/_index.es.md]: What you wrote in Spanish literally would mean:

"None of your business!" 😅

Of you want to preserve the original meaning of "Why it matters", although this construction is not as common in Spanish I suppose you could use:

¿Por qué tiene tanta importancia?

Comment (arschles) [reply docs/content/_index.es.md]: >What you wrote in Spanish literally would mean:
>
>"None of your business!" 😅

HAHAHA! Sounds like what you wrote is 1000x better then

File: docs/content/_index.es.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (arschles) [reply docs/content/_index.es.md]: Deseando is great then!

**Comment by arschles** [comment]
@lopezator let me know when https://github.com/gomods/athens/pull/1134#discussion_r266808507 is ready and I'll :shipit:

File: docs/content/_index.es.md (hunk 1 above)
Comment (lopezator) [reply docs/content/_index.es.md]: Changed by: `¿Por qué es tan importante?` which literally means "Why is so important?" but makes more sense in spanish IMHO.

**Comment by lopezator** [comment]
Done. PTAL @arschles

**Pull request merged**